reference/employer/openapi.yaml

```diff
@@ -926,6 +932,288 @@ paths:
       tags:
         - Management
+  /employer/company/benefits:
+    get:
+      summary: List Company Benefits
+      tags:
+        - Employer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    benefit_id:
+                      type: string
+                    type:
+                      type: string
+                    description:
+                      type: string
+                    frequency:
+                      type: string
+                    company_contribution:
+                      type: string
+                    employee_deduction:
+                      type: string
+      operationId: get-company-benefits
+      description: List all company-wide benefits.
+    post:
+      summary: Create Company-wide Benefit
+      operationId: post-company-benefits
+      responses:
+        '201':
+          description: Successfully Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  benefit_id:
+                    type: string
+                  description:
+                    type: string
+      description: Creates a new company-wide benefit
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                type:
+                  type: string
+                description:
+                  type: string
+                employee_deduction:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                    amount:
+                      type: number
+                company_contribution:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                    amount:
+                      type: integer
+                frequency:
+                  type: string
+                annual_maximum:
+                  type: string
+      tags:
+        - Employer
+  /employer/company/benefits/meta:
+    get:
+      summary: List Benefits Metadata
+      tags:
+        - Employer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                    description:
+                      type: string
+                    frequencies:
+                      type: array
+                      items:
+                        type: string
```

Comment:
Rather than note everywhere with `enum` I'll just comment this one time. There are some more places above too for other fields like `type`.